Src/StdLib/Lib/test/test_iterlen.py

```diff
@@ -148,6 +148,7 @@ class TestList(TestInvariantWithoutMutations, unittest.TestCase):
     def setUp(self):
         self.it = iter(range(n))
 
+    @unittest.skip("Test crashing test suite")
```

Comment:
Probably best to put a comment to indicate the skip is specifically for ironpython. Makes it easier to sync the StdLib files with CPython.